suite-native/transactions/src/components/TransactionDetail/TransactionDetailInputsSheet.tsx

```diff
@@ -1,110 +1,97 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
 
-import { NetworkSymbol } from 'suite-common/wallet-config/src';
+import { G } from '@mobily/ts-belt';
 
-import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 import { Icon } from '@suite-common/icons';
-import { WalletAccountTransaction } from '@suite-common/wallet-types';
-import { Box, Card, Text, VStack } from '@suite-native/atoms';
-import { AccountAddressFormatter, CryptoAmountFormatter } from '@suite-native/formatters';
+import { AccountKey } from '@suite-common/wallet-types';
+import { Box, Text, VStack } from '@suite-native/atoms';
+import { FiatRatesRootState } from '@suite-native/fiat-rates';
+import { TransactionsRootState } from '@suite-common/wallet-core';
+import { SettingsSliceRootState } from '@suite-native/module-settings';
 
 import { TransactionDetailSheet } from './TransactionDetailSheet';
+import { selectTransactionInputAndOutputTransfers } from '../../selectors';
+import { TransactionDetailInputsSheetSection } from './TransactionDetailInputsSheetSection';
 
 type TransactionDetailInputsSheetProps = {
     isVisible: boolean;
-    transaction: WalletAccountTransaction;
+    txid: string;
+    accountKey: AccountKey;
     onSheetVisibilityChange: () => void;
 };
 
-type TransactionAddressAmountProps = {
-    address: string;
-    amount?: string;
-    symbol: NetworkSymbol;
+type InputsOutputsMainHeader = {
```

Comment:
It's weird that eslint is not screaming here because you're having `InputsOutputsMainHeader` declaration twice in this file

```suggestion
type InputsOutputsHeaderProps = {
```

Reply:
It did not shout because one of them was a type and another a const. But indeed it was not correct. Fixed in 70e3a7bc9.

Reply:
That's weird. I get this when I have same name of type and component props. 

<img width="1020" alt="image" src="https://github.com/trezor/trezor-suite/assets/36101761/aa19129b-049c-41b7-88db-10c9b00030e1">
<img width="840" alt="image" src="https://github.com/trezor/trezor-suite/assets/36101761/aec79b3a-eafb-43e2-966a-6803c976fcde">

Reply:
The error says, that DevXPub is never used. Not that you are redeclaring the const.